HHS/OPRE-OPS
# Refine CANs data model

## What changed

See tasks in issue description!

## Issue
#190 

## How to test locally

- Pull down branch
- Spin up app locally
- Navigate to `CANs` list on localhost djang admin
- See new fields/new order
- If you click into one of the rows you can see that CAN leads are now a many to many field (no restrictions on who can be a CAN lead).


## Screenshots
<img width="1406" alt="Screen Shot 2021-09-27 at 3 00 29 PM" src="https://user-images.githubusercontent.com/10871517/134984605-20f65c90-cc5c-457f-944f-788ee87484fd.png">
<img width="1427" alt="Screen Shot 2021-09-27 at 3 00 38 PM" src="https://user-images.githubusercontent.com/10871517/134984611-a98a5a77-7142-49fd-9273-aca1ec7d899c.png">
